packages/react/src/useUploadThing.ts

```diff
@@ -134,7 +135,11 @@ export const INTERNAL_uploadthingHookGen = <
     return {
       startUpload,
       isUploading,
+      /**
+       * @deprecated Use `routeConfig` instead
+       */
       permittedFileInfo,
```

Comment:
the `permittedFileInfo: { slug: string, config: ExpandedRouteConfig }` is just an "annoyance" - you only care about the config part. the `slug` is what you put into the `useUploadThing(slug, opts)` anyways...